doc/install-dependencies.md

```diff
@@ -25,17 +25,17 @@
 - [Ubuntu 16.04 & 18.04](#ubuntu-1604--1804)
 - [Debian 9 & 10](#debian-9--10)
 - [Mac OSX](#mac-osx)
-- [How to compile the Openresty](#how-to-compile-the-openresty)
+- [How to compile the OpenResty](#how-to-compile-the-openresty)
 
 CentOS 6
 ========
 
 ```shell
-# add openresty source
```

Comment:
@moonming  Shoud I change `openresty` to `OpenResty`

Reply:
or only `Openresty`?

Reply:
`OpenResty` is fine.